lib/internal/source_map/source_map_cache.js

```diff
@@ -171,7 +171,16 @@ function sourcesToAbsolute(base, data) {
     if (!/^[\\/]/.test(source[0])) {
       source = resolve(base, source);
     }
-    if (!source.startsWith('file://')) source = `file://${source}`;
+    if (!source.startsWith('file://')) {
```

Comment:
```suggestion
    if (!StringPrototypeStartsWith(source, 'file://')) {
```

Reply:
This file does not import ``StringPrototypeStartsWith``. I have added it from ``primordials`` and replaced it in one more location.

Reply:
Thanks, much appreciated :-)